source/_integrations/generic_thermostat.markdown

```diff
@@ -69,7 +69,7 @@ hot_tolerance:
 keep_alive:
   description: Set a keep-alive interval. If set, the switch specified in the *heater* option will be triggered every time the interval elapses. Use with heaters and A/C units that shut off if they don't receive a signal from their remote for a while. Use also with switches that might lose state. The keep-alive call is done with the current valid climate integration state (either on or off).
   required: false
-  type: [time, integer]
+  type: time
```

Comment:
This change is not correct, it accepts both time or an integer (which will be interpreted as seconds).

Reply:
same as above